### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by steelbrain**
- Upgrade dependencies

### CODE REVIEW

**Comment by steelbrain** [comment]
I'm gonna try to use this in a project to determine if this should be a major or minor release bump

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+#### 2.0.0
+
+- Upgrade to ESLint v3.5.0 to include bugfixes
+- Upgrade to AirBnb Base Config v7.1.0
```
Comment (Arcanemagus) [inline CHANGELOG.md]: Apparently they prefer it as `Airbnb` instead of `AirBnb` btw.

**Review by Arcanemagus — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (steelbrain) [reply CHANGELOG.md]: Updated, Thanks 👍

**Comment by Arcanemagus** [comment]
As `eslint-config-airbnb-base` is (several) major updates in here, and it's the basis for this style, technically it should be a breaking change. In practice I found only the latest v7.1.0 of it really needed any changes.

**Review by Arcanemagus — APPROVED** [review]

**Pull request merged**